apps/docs/src/components/AIPanel/AdminImprovementReview.tsx

```
@@ -0,0 +1,247 @@
+                  <strong>Document:</strong>{" "}
+                  <a href={improvement.docId} target="_blank" rel="noopener noreferrer">
+                    {improvement.docId}
+                  </a>
+                </div>
+
+                <div className="admin-review-suggestions">
+                  <h4>AI Suggestions:</h4>
+                  <div
+                    className="admin-review-suggestions-content"
+                    dangerouslySetInnerHTML={{
+                      __html: formatMarkdown(improvement.suggestions),
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)